
This reverts:
---
From 2fdb48fa3333638cee889b8bb80dc1d2b65aaa4a Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 16 Jan 2018 10:35:27 +0100
Subject: [PATCH] power: Default to suspend after 20 minutes of inactivity
---

as the inactivity checking in Gnome is broken, causing systems to shut down
even if users are logged in either over ssh or in a multi-seat setup screwing
up things and even cause data corruption or data loss..

can be restored when Gnome is properly fixed...

Signed-off-by: Thomas Backlund <tmb@mageia.org>

diff -Nurp gnome-settings-daemon-3.29.90.1.orig/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in gnome-settings-daemon-3.29.90.1/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in
--- gnome-settings-daemon-3.29.90.1.orig/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in	2018-08-01 22:23:08.000000000 +0300
+++ gnome-settings-daemon-3.29.90.1/data/org.gnome.settings-daemon.plugins.power.gschema.xml.in	2018-08-24 19:39:39.298223207 +0300
@@ -12,7 +12,7 @@
       <description>If the screen should be dimmed to save power when the computer is idle.</description>
     </key>
     <key name="sleep-inactive-ac-timeout" type="i">
-      <default>1200</default>
+      <default>0</default>
       <summary>Sleep timeout computer when on AC</summary>
       <description>The amount of time in seconds the computer on AC power needs to be inactive before it goes to sleep. A value of 0 means never.</description>
     </key>
@@ -22,7 +22,7 @@
       <description>The type of sleeping that should be performed when the computer is inactive.</description>
     </key>
     <key name="sleep-inactive-battery-timeout" type="i">
-      <default>1200</default>
+      <default>0</default>
       <summary>Sleep timeout computer when on battery</summary>
       <description>The amount of time in seconds the computer on battery power needs to be inactive before it goes to sleep. A value of 0 means never.</description>
     </key>
